src/components/Dashboard/FilingHistoryList/StaffFiling.vue

```diff
@@ -2,6 +2,17 @@
   <div v-if="filing" class="staff-filing-details body-2">
     <p>{{filing.notationOrOrder}}</p>
 
+    <template v-if="filing.documents && filing.documents.length > 0">
+      <DocumentsList
+                :filing=filing
+                :loadingOne=loadingOne
+                :loadingAll=loadingAll
+                :loadingOneIndex=loadingOneIndex
+                @downloadOne="downloadOne(...arguments)"
+                @downloadAll="downloadAll($event)"
+              />
```

Comment:
Too much indenting.

Reply:
Do we have a standardized format in VSCode to avoid problems like this?

Reply:
There may be tools that indent the template (HTML) automatically and consistently. Do you have a recommendation?

For the script (TS/JS) (and CSS too I think), the linter catches these.